settings/server/config.default.js

```diff
@@ -1,3 +1,3 @@
 module.exports = {
-    port: 
+    port:
```

Comment:
Help me change this file name to 'config.js.default',
in that way eslint won't be buged and complain about this file.